packages/graph/hash_graph/lib/graph/src/types/schema/property_type.rs

```diff
@@ -193,13 +198,15 @@ mod tests {
         property_type: &PropertyType,
         uris: impl IntoIterator<Item = &'static str>,
     ) {
-        let expected_property_references = uris.into_iter().map(Uri::new).collect::<HashSet<_>>();
+        let expected_property_references = uris
+            .into_iter()
+            .map(ToString::to_string)
+            .collect::<HashSet<_>>();
 
         let property_references = property_type
             .property_type_references()
             .into_iter()
-            .map(PropertyTypeReference::reference)
-            .cloned()
+            .map(|reference| reference.uri().base_uri().to_string())
```

Comment:
Why are the `data_type_references` only using the `::uri` method but the `property_references` uses `base_uri`? 🤔